app/models/delivery_order_importer.rb

```diff
@@ -0,0 +1,20 @@
+class DeliveryOrderImporter < ActiveImporter::Base
+  imports DeliveryOrder
+
+  skip_rows_if { row['GUIA_INTERNA'].blank? }
+
+  column 'FECHA', :radication_at
+  column 'N°_CARGUE', :charge_number
+  column 'MENSAJERO', :delivery_man
+  column 'CIUDAD', :city
+  column 'GUIA_INTERNA', :internal_guide
+  column 'DESTINATARIO', :destinatary
+  column 'DIRECCION', :adderss
+  column 'CLIENTE', :client
+  column 'GUIA_EXTERNA', :externa_guide
+
+  on :import_failed do |ex|
+    Rails.logger.error("Could not start importing data: #{ex}")
```

Comment:
Manejemos un formato de error que incluya el nombre de la clase donde ocurre el error, algo como Rails.logger.error("DeliveryOrderImporter -> Could not start importing data: #{ex}")